ssas/service/public/middleware.go

```diff
@@ -44,29 +50,30 @@ func readGroupID(next http.Handler) http.Handler {
 // occurs in requireRegTokenAuth() or requireMFATokenAuth().
 func parseToken(next http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		event := ssas.Event{Op: "ParseToken"}
+		event := logrus.Fields{"Op": "ParseToken"}
+		logger := log.GetCtxLogger(r.Context()).WithFields(event)
 		authHeader := r.Header.Get("Authorization")
 		if authHeader == "" {
-			event.Help = "no authorization header found"
-			ssas.AuthorizationFailure(event)
+			helpMsg := "no authorization header found"
+			logger.Error(logrus.Fields{"Event": "AuthorizationFailure", "Help": helpMsg})
```

Comment:
For this line, when passing in a logrus.Fields{} map, do those fields get automatically added to the log message the same way they do when calling `logger.WithFields`? Just wondering if putting them in the log level statement will add them to the `msg` field instead of creating `Event` and `Help` as their own fields.

As a side note, the helpMsg should probably go in the `msg` field instead of creating a new field.